README.md

```diff
@@ -87,3 +87,7 @@ Explore this repositories commit history to look at your changes and learn about
 **Revert a commit**
 
 &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;&nbsp;`git revert <commit hash>`
+
+**Save your username and password in git (Warning! This involves a potential security risk!)**
```

Comment:
```suggestion

```

Reply:
Line 91 is meant to be a description for line 92... how about rewording it some way different?

Reply:
Sure. For consistency, try to format it similarly to the other tool tips though.